packages/fether-electron/src/main/app/menu/template/index.js

```diff
@@ -156,7 +156,7 @@ const getMenubarMenuTemplate = fetherApp => {
 
 const getContextTrayMenuTemplate = fetherApp => {
   if (fetherApp.options.withTaskbar) {
-    const template = [
+    let template = [
```

Comment:
Should be `const` here, the array is mutable, but the reference is not.

Reply:
@amaurymartiny I've pushed a commit to address your comment